enterprise/frontend/src/metabase-enterprise/license/components/LicenseAndBillingSettings/LicenseAndBillingSettings.tsx

```diff
@@ -97,48 +116,21 @@ const LicenseAndBillingSettings = ({
     );
   }
 
-  const isInvalid = !!error || (tokenStatus != null && !tokenStatus.isValid);
-  const description = getDescription(tokenStatus, !!token);
+  const hasToken = !!token || is_env_setting;
```

Comment:
noticed we were showing text for the user to enter a token when there was one set in the env & the input was disabled due to there being a token in the env